components/VM_Arm/CMakeLists.txt

```diff
@@ -127,6 +127,16 @@ config_option(
     OFF
 )
 
+config_option(
+    VmVirtUart
+    VM_VIRT_UART
+    "Option for virtualizing console"
+    DEFAULT
+    OFF
+    DEPENDS
+    "KernelPlatformExynos5410 OR KernelPlatformExynos5422 OR  KernelPlatformZynqmp"
```

Comment:
```suggestion
    "KernelPlatformExynos5410 OR KernelPlatformExynos5422 OR KernelPlatformZynqmp"
```

Reply:
Ah dang, I hate when there's extra whitespace changes like that. I'll fix it.